packages/forms/src/UIForm/Widget/condition.js

```diff
@@ -1,18 +1,81 @@
 import get from 'lodash/get';
-import includes from 'lodash/includes';
+import findIndex from 'lodash/findIndex';
 
-function getValue(value, strategy) {
-	if (!strategy) {
+function toNumber(value) {
+	if (typeof value === 'string') {
+		return Number(value);
+	}
+	return value;
+}
+
+function toString(value) {
+	if (!value) {
 		return value;
 	}
-	switch (strategy.toLowerCase()) {
+	if (typeof value === 'number') {
+		return JSON.stringify(value);
+	}
+	if (value.toString) {
```

Comment:
Wouldn’t it be better tonremove this check and return ´toString(value)’ so the function will consistantly return a string ?

Reply:
not sure I got it, it will loop if I call it again here and it is not a number, i can likely throw an exception if not a string instead of doing this check which is quite poor for objects